benchmarks/fasta/php/bench.php

```diff
@@ -10,7 +10,8 @@
 ob_implicit_flush(1);
 ob_start(NULL, 4096);
 
-$last = 42.0;
+$INITIAL_STATE = 42.0;
```

Comment:
Nitpicking here, but you can use `define` to define constants in PHP. I don't mind if you leave it either.